cylc/flow/scripts/cylc.py

```diff
@@ -431,7 +429,10 @@ def cli_help():
     from colorama import init as color_init
     color_init(autoreset=True, strip=False)
     print(USAGE)
-    print('Selected Sub-Commands:')
+    print(
+        'Selected sub-commands '
+        '(type "cylc help all" to see ALL sub-commands):'
+    )
```

Comment:
I feel the `cylc help all` bit was more prominent before when it was at the end of the list?

I.E. User scans down the list, can't find what they're looking for, ah `cylc help all`.

Reply:
Yep, you might be right there.